frontend/test/metabase/scenarios/sharing/public.cy.spec.js

```diff
@@ -21,10 +22,7 @@ const USERS = {
   "anonymous user": () => cy.signOut(),
 };
 
-// [quarantine]: failing almost consistently in CI
-// Skipping the whole spec because it needs to be refactored.
-// If possible, re-use as much code as possible but let test run in isolation.
-describe.skip("scenarios > public", () => {
+describe("scenarios > public", () => {
```

Comment:
@nemanjaglumac this was skipped in https://github.com/metabase/metabase/pull/13945 which is almost 2 years ago, can you help review if this is good now or should it still be disabled?

I tested locally > 10 times and it's still going strong in green.